packages/paste-core/components/toast/__tests__/useToaster.spec.tsx

```diff
@@ -119,7 +122,7 @@ describe('useToaster', () => {
       act(() => {
         result.current.pop('custom_timeout');
       });
-      expect(clearTimeout).toHaveBeenCalled();
+      expect(clearTimeoutSpy).toHaveBeenCalled();
```

Comment:
This is a required update on how to spy on window methods in the new version of Jest